internal/controller/state/graph/grpcroute.go

```diff
@@ -260,6 +265,12 @@ func processGRPCRouteRules(
 	conds = make([]conditions.Condition, 0, 2)
 	valid = true
 
+	// add warning condition for unsupported fields if any
+	if len(allRulesErrors.warn) > 0 {
+		msg := "There are rules with unsupported fields configurations: " + allRulesErrors.warn.ToAggregate().Error()
+		conds = append(conds, conditions.NewRouteUnsupportedField(msg))
```

Comment:
Sorry, one more thing - I deployed locally and the message has too many redundant fields:

```Last Transition Time:  2025-10-08T13:09:29Z
      Message:               Some parameters are ignored because they are not supported: There are rules with unsupported fields configurations: spec.rules[0].name: Forbidden: rule names are not supported
      Observed Generation:   2
      Reason:                UnsupportedField
      Status:                True
      Type:                  UnsupportedField
```
We should remove the `There are rules with unsupported fields configurations: ` message prefix.

We can simplify the whole thing then to

```
	if len(allRulesErrors.warn) > 0 {
		conds = append(conds, conditions.NewRouteUnsupportedField(allRulesErrors.warn.ToAggregate().Error()))
	}
```